src/xo-app/host/tab-network.js

```diff
@@ -180,30 +302,29 @@ class PifItem extends Component {
           disabled={pif.attached && (pif.management || pif.disallowUnplug)}
           handlerParam={pif}
           state={pif.attached}
-        />
+              />
```

Comment:
Why this change?

Reply:
I fixed it by taking the red line and removing the green one :)